spikeinterface/sorters/runsorter.py

```diff
@@ -232,9 +250,14 @@ def __init__(self, mode, container_image, volumes, extra_kwargs):
                 client.images.pull(container_image)
 
             self.docker_container = client.containers.create(
-                    container_image, tty=True, volumes=volumes, **extra_kwargs)
+                container_image,
+                tty=True,
+                volumes=volumes,
+                **extra_kwargs
+            )
 
         elif mode == 'singularity':
+            assert self.py_user_base, 'py_user_base folder must be set in singularity mode'
```

Comment:
Maybe it would also make sense to raise a warning in case the provided directory is overlapping with a default / system python package location?

Reply:
This location is not provided by the user. It's a folder we create internally as `in_container_python_base`
Also there are a few locations that python checks by default and probably OS dependent, I'm not sure how to check for this properly